plugins/scaffolder-backend/src/scaffolder/actions/builtin/publish/githubPullRequest.ts

```diff
@@ -219,56 +217,39 @@ export const createPublishGithubPullRequestAction = ({
         ? resolveSafeChildPath(ctx.workspacePath, sourcePath)
         : ctx.workspacePath;
 
-      const localFilePaths = await globby(['./**', './**/.*', '!.git'], {
-        cwd: fileRoot,
+      const directoryContents = await serializeDirectoryContents(fileRoot, {
```

Comment:
Do we need to filter out the `.git` and does this also include hidden files and folders?